core/templates/domain/android-updates/android-updates-backend-api.service.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Backend api service for android email updates
+ */
+
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class AndroidUpdatesBackendApiService {
+  constructor(private http: HttpClient) {}
+
+  private async _putRequestAsync(
+      handlerUrl: string, payload: Object): Promise<void> {
```

Comment:
Can we specify a interface for the payload?

Reply:
Done